src/SFA.DAS.EmployerApprenticeshipsService.Application.UnitTests/Commands/CreateAccountCommandTests/WhenICallCreateAccount.cs

```diff
@@ -36,9 +37,10 @@ public class WhenICallCreateAccount
         private Mock<IGenericEventFactory> _genericEventFactory;
         private Mock<IAccountEventFactory> _accountEventFactory;
         private Mock<IRefreshEmployerLevyService> _refreshEmployerLevyService;
+        private Mock<IMembershipRepository> _mockMembershipRepository;
         private const long ExpectedAccountId = 12343322;
         private const long ExpectedLegalEntityId = 2222;
-        private const string ExpectedHashString = "123ADF23";
+        private const string ExpectedHashString = "1234hash";

```

Comment:
just out of curiosity why was this value changed?

Reply:
I can only imagine that I had flip flopped from accountId to hashedAccountId and changed the string. It seems like an odd thing to do though admittedly and I can't see why I would have done it. A moment of complete madness.

Reply:
I've set it back to what it was.